examples/2d/sprite_scale.rs

```diff
@@ -5,10 +5,7 @@ use bevy::prelude::*;
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
-        .add_systems(
-            Startup,
-            (setup_sprites, setup_texture_atlas).after(setup_camera),
```

Comment:
do we know the original rationale behind this part if there was one?

Reply:
Tt was introduced in the original PR to add the example, and seems to have just been a mistake that was overlooked.

https://github.com/bevyengine/bevy/commit/deb135c25c3a7eafdc432eaefe0957d6ffdcca62